docs/source/basics/parameter_types.md

```diff
@@ -11,18 +11,36 @@ registration.
 
 ## Supported Types
 
-Below is an exhaustive list of supported types currently shipped with the SDK.
+Tasks support all native Python types, while workflow only supports a subset of types.
```

Comment:
this is false? we certainly don't support "all native Python types" by pretty much any defnition

also the workflows and tasks support the same stuff